src/gates.jl

```diff
@@ -945,6 +1025,48 @@ function CRZGate(θ::Number)
     return round_complex_values(CRZ)
 end
 
+@doc raw"""
+    CRZGate(θ::Number)
```

Comment:
Shouldn't this be CRZRevGate?

Reply:
And, does this affect any accuracy of the tests?

Reply:
Fixed